pkg/controllers/user/helm/common/common.go

```diff
@@ -169,7 +204,7 @@ func getTimeoutArgs(app *v3.App) []string {
 	if app.Spec.Wait {
 		timeoutArgs = append(timeoutArgs, "--wait")
 	}
-	timeoutArgs = append(timeoutArgs, "--timeout", strconv.Itoa(app.Spec.Timeout))
+	timeoutArgs = append(timeoutArgs, "--timeout", fmt.Sprintf("%ss", strconv.Itoa(app.Spec.Timeout)))
```

Comment:
The double s is for seconds ?

Reply:
Was this a bug before? It doesn't seem we are distinguishing between helm versions where this is used